low-level/framework/sspl_ll_d

```diff
@@ -853,15 +853,17 @@ if __name__ == "__main__":
                     print("Another instance of SSPL with pid %s is active. exiting..." % pid)
                     sys.exit(os.EX_OK)
 
+            if not os.path.isdir('/var/run/sspl_ll/'):
+                os.mkdir('/var/run/sspl_ll/')
+
             with open(pidfile, "w") as fileObj:
                 fileObj.write(str(os.getpid()))
 
         # Start sspl-ll as a main process running multiple threads
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
-        traceback.print_exc()
-        logger.critical("While spawning sspl-ll process :%r" % err)
+        logger.exception("Error while initiliazing sspl-ll service: ")
```

Comment:
Need to print err. We can not simply fail without any meaningful information.

Reply:
@ujjwalpl this is the generic exception in sspl_ll_d main, which would come for any unhandled exception anywhere in sspl, so adding specific error is hard to add. logger.exception would give us required details from code to identify though

Reply:
@ujjwalpl recommending pin-point pattern instead of this big-tarp pattern. Though am saying we dont have specific exceptions defined everywhere, so this generic big-tarp exception pattern would help to log enough details on facing exception.
Its same what we mentioned above, just with pattern names now.
https://aviadr1.github.io/learn-advanced-python/15_logging/logging_in_python.html#the-pinpoint-pattern

Reply:
We dont want to lose err. It is important to print the reason so its easy to know why SSPL fails to start. 
Please incorporate and print err. This is an easy change.

Reply:
Sorry.. I could not understand,
Do I need to add   **logger.exception(f"Error while initiliazing sspl-ll service:  {err}")**
But exception info (here whatever captured in err)  this will automatically gets added, in logger.exception(), r8?

Reply:
logger.exception() would print err too

Reply:
@shriya-deshmukh please incorporate below exception handing code, concluded along with @ujjwalpl 
> logger.exception("While spawning, sspl-ll process encountered critical error : %s" %e)

Reply:
Done.